src/modules/ContentList/index.tsx

```diff
@@ -289,6 +217,79 @@ export function ContentList(_props: ContentListProps): ReactElement {
         'in'
     );
 
+    const contentListQuery = useQuery<ContentListQuery, ContentListQueryVariables>({
+        isEmpty: ({ content }) => (content || []).length === 0,
+        query: gql`
+            query ContentList($filter: ContentFilter!) {
+                content(filter: $filter) {
+                    id
+                    contentType
+                    primaryCompany {
+                        id
+                        commonName
+                        instruments {
+                            id
+                            isPrimary
+                            quotes {
+                                id
+                                isPrimary
+                                localTicker
+                                exchange {
+                                    id
+                                    shortName
+                                    country {
+                                        id
+                                        countryCode
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    publishedDate
+                    source
+                    title
+                }
+            }
+        `,
+        requestPolicy: 'cache-and-network',
+        variables: {
+            filter: {
+                contentIds: [
```

Comment:
What are these ids?

Reply:
Placeholder. Right now the `content` query requires a filter with `contentIds`. I added a comment with a TODO to replace this query with the new search api. The `Content` module will continue to use this `content` query.